gui/wxpython/lmgr/workspace.py

```diff
@@ -343,6 +343,15 @@ def Load(self, filename):
                 self.lmgr.nvizUpdateSettings()
                 mapdisplay[i].toolbars["map"].combo.SetSelection(1)
 
+        #
+        # load layout
+        #
+        if UserSettings.Get(group="general", key="singleWindow", subkey="enabled"):
+            if gxwXml.layout["panes"]:
+                self.lmgr.GetAuiManager().LoadPerspective(gxwXml.layout["panes"])
+            if gxwXml.layout["notebook"]:
+                self.lmgr.GetAuiNotebook().LoadPerspective(gxwXml.layout["notebook"])
+
```

Comment:
For backwards compatibility (reading a workspace file created before this PR) you need to check whether attribute layout exists.

Reply:
I think that this is not needed, starting this PR, the self.layout will always be defined:
```
        self.layout = {
            "panes": None,
            "notebook": None,
        }
```
when I open a workspace file where parameter layout does not exist, the dictionary self.layout remains with None values.
I tested the loading of previously created workspaces files and it works fine.